renderdoc/os/win32/win32_stringio.cpp

```diff
@@ -166,6 +167,11 @@ void GetExecutableFilename(string &selfName)
   selfName = StringFormat::Wide2UTF8(wstring(curFile));
 }
 
+bool IsRelativePath(const string &path)
+{
+  return PathIsRelativeA(path.c_str()) != 0;
```

Comment:
Whoops, I did that one too late at night, didn't look at surrounding calls.  Fixed.